E2E-tests/config/substrate/staging_nodes.json

```diff
@@ -143,6 +143,11 @@
                 "lock": 817414141,
                 "send": 298945143
             }
-        }
+        },
+        "permissioned_min_ariadne_d_param_value": 1,
```

Comment:
Same as above. These values can stay the same on staging, and the test can be done on the local cluster, IIRC.